planex/cmd/mock.py

```diff
@@ -91,8 +91,9 @@ def mock(args, tmp_config_dir, *extra_params):
     if args.resultdir is not None:
         cmd += ["--resultdir", args.resultdir]
 
-    for define in args.define:
-        cmd += ['--define', " ".join(define)]
+    for key, value in args.define:
+        if key != "_topdir":
```

Comment:
I know that it's already in the commit message, but maybe it's worth adding the reason of this check as a comment as well

Reply:
👍